app/views/licence/_authority_details.html.erb

```diff
@@ -38,7 +63,19 @@
     <% if licence_details.local_authority_specific? or licence_details.multiple_licence_authorities_present? %>
       <div class="contact">
         <p class="govuk-body">The issuing authority for this licence is <strong><%= licence_details.authority["name"] %></strong>
-          <%= link_to (licence_details.local_authority_specific? ? '(change location)' : '(change authority)'), licence_path(publication.slug), class: "govuk-link" %>
+          <%= link_to (licence_details.local_authority_specific? ? '(change location)' : '(change authority)'), 
+            licence_path(publication.slug),
+            class: "govuk-link",
+            data: {
+              ga4_link: {
+                event_name: "form_change_response",
+                action: "change response",
+                type: content_item_hash["schema_name"].gsub("_", " "),
+                tool_name: publication.title
+              },
+              ga4_do_not_index: ""
```

Comment:
I understand this now (thanks) but maybe put a comment here to briefly explain, something like `don't give this link an index (override behaviour of data-ga4-set-indexes on the parent)`